src/main/java/doit/apihw/api/controller/MemberController.java

```diff
@@ -51,5 +46,11 @@ public void changePassword(@PathVariable Long memberId, @RequestBody AuthPasswor
     }
 
     // TODO : 자유 주제로 API를 추가로 구현해보세요.
-
+    /**
+     * 회원 탈퇴 시 회원 정보를 삭제한다.
+     */
+    @DeleteMapping("/members/{memberId}/delete")
+    public void deleteMember(@PathVariable Long memberId) {
+        memberService.deleteMember(memberId);
+    }
```

Comment:
Delete Method 사용해주시는 부분 RESTful하게 좋습니다.
저도 종종 delete 관련 메소드 짤 때, 아래처럼 짜기도 하는데 아래처럼 짜면 api 형식이 깔끔해져서 좋더라구요. 참고해보시면 좋을 것 같습니다.
```suggestion
    /**
     * 회원 탈퇴 시 회원 정보를 삭제한다.
     */
    @DeleteMapping("/members/{memberId}")
    public void deleteMember(@PathVariable Long memberId) {
        memberService.deleteMember(memberId);
    }
```